gnomad_qc/v2/resources/basics.py

```diff
@@ -507,10 +515,6 @@ def vep_config_path(ref: str = 'GRCh37'):
 
 
 # Sample QC files
-def qc_mt_path(data_type: str):
-    return 'gs://gnomad/sample_qc/mt/gnomad.{}.high_callrate_common_biallelic_snps.mt'.format(data_type)
-
-
 def qc_ht_path(data_type: str):
```

Comment:
why does this path exist in both `basics` and `sample_qc.py`?

Reply:
not sure... the history of these files are quite complex. Clearly it makes no sense though.